odata-url-conventions/5 Query Options.md

```diff
@@ -2096,11 +2073,11 @@ http://host/service/Categories?$expand=Products/$count($search=blue)
 
 To retrieve entity references instead of the related entities, append
 `/$ref` to the navigation property name or [type-cast segment](#AddressingDerivedTypes) following a navigation property name.
-The system query options [`$filter`](#SystemQueryOptionfilter),
+The [Expand Options](#ExpandOptions) [`$filter`](#SystemQueryOptionfilter),
 [`$search`](#SystemQueryOptionsearch),
 [`$skip`](#SystemQueryOptionstopandskip),
 [`$top`](#SystemQueryOptionstopandskip), and
-[`$count`](#SystemQueryOptioncount) can be used to limit the number of
+[`$count`](#SystemQueryOptioncount) can be used to include the count of
 expanded entity references.
```

Comment:
```
The [Expand Options](#ExpandOptions) [`$filter`](#SystemQueryOptionfilter),
[`$search`](#SystemQueryOptionsearch),
[`$skip`](#SystemQueryOptionstopandskip),
[`$top`](#SystemQueryOptionstopandskip) can be used to limit the collection of expanded entity references, and
[`$count`](#SystemQueryOptioncount) can be used to include not the entity references themselves
but their count.
```

Reply:
I wonder at the advantage of /$ref/$count over just /$count.  Should we support the former?

Reply:
sorry; my bad -- this is not /$ref/$count, this is /$ref($count=true).  we actually have abnf test cases verifying that /$ref/$count fails.